### PR TITLE
Remove a git-hooks check that is not applicable to the repo

### PR DESCRIPTION
**Pull request by rene-dekker**

### CODE REVIEW

**Review by tmjd — APPROVED** [review]
LGTM

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Pull request merged**
